src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -3254,6 +3254,59 @@ public static int indexOf(final Object[] array, final Object objectToFind, int s
         return INDEX_NOT_FOUND;
     }
 
+    /**
+     * <p>Finds the indices of the given object in the array.
```

Comment:
Oops, I did not notice before that you did not close any of these HTML tags. You need to close paragraphs.

Reply:
@garydgregory Fixed. Apparently this is a problem with other functions too